pkg/commands/compute/language_go.go

```diff
@@ -85,33 +89,64 @@ type Go struct {
 	spinner text.Spinner
 	// timeout is the build execution threshold.
 	timeout int
+	// toolchain indicates the toolchain used for compiling your program.
+	toolchain string
 	// verbose indicates if the user set --verbose
 	verbose bool
 }
 
 // Build compiles the user's source code into a Wasm binary.
 func (g *Go) Build() error {
-	var noBuildScript bool
+	var (
+		tinygoToolchain     bool
+		toolchainConstraint string
+	)
+
 	if g.build == "" {
-		g.build = GoDefaultBuildCommand
-		noBuildScript = true
+		g.build = TinyGoDefaultBuildCommand
+		tinygoToolchain = true
+		toolchainConstraint = g.config.ToolchainConstraintTinyGo
+		text.Info(g.output, "No [scripts.build] found in fastly.toml. Visit https://developer.fastly.com/learning/compute/go/ to learn how to target native Go vs TinyGo.")
```

Comment:
Is `native Go` really what we want to call this compiler option?

Reply:
Can we use `standard Go` ?

Reply:
👍🏻